Workflow failure: Admin UI E2E - realm_settings_user_profile_enabled

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Some realm_settings_user_profile_enabled test failed during the run
https://github.com/keycloak/keycloak/actions/runs/8073144290/job/22056636796#step:11:156

`User profile tabs
       Check attributes are displayed and editable on user create/edit
         Checks that not required attribute is not present when user is created/edited with email as username enabled:
     AssertionError: Timed out retrying after 30000ms: Expected to find element: `.pf-c-alert__title`, but never found it. Queried from:`


### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/js/apps/admin-ui/cypress/e2e/realm_settings_user_profile_enabled.spec.ts
+++ b/js/apps/admin-ui/cypress/e2e/realm_settings_user_profile_enabled.spec.ts
@@ -386,7 +386,7 @@ describe("User profile tabs", () => {
     });
 
     it("Checks that attribute with select-annotation is displayed and editable when user is created/edited", () => {
-      const userName = "select-test-user";
+      const userName = `select-test-user-${uuid()}`;
       const attrName = "select-test-attr";
       const opt1 = "opt1";
       const opt2 = "opt2";

